workflow/artifacts/http/http.go

```diff
@@ -17,6 +18,11 @@ type HTTPArtifactDriver struct{}
 func (h *HTTPArtifactDriver) Load(inputArtifact *wfv1.Artifact, path string) error {
```

Comment:
at some point I'd love to replace `curl` with native Golang code - faster, more secure and less complex - like the artifactory artifact does - but out of scope for this PR

Reply:
Interesting - are you sure it's faster though? I thought libcurl was pretty quick...

Reply:
well, not today